internal/cmd/access/access.go

```diff
@@ -80,6 +81,10 @@ func runAccess(cfg *accessConfig) clierror.Error {
 		fmt.Println(string(message))
 
 	}
+	err = kube.SaveConfig(enrichedKubeconfig, cfg.output)
```

Comment:
Do we save config twice (first time in rows 70-83, second in 84-87)?

Reply:
I forgot to remove it from this command, fixed